### ISSUE TITLE
Rails 2.x support?

### ISSUE THREAD

**Issue by dnrce** [yamldb/yaml_db#69]
(Meta-issue) Now that yaml_db has returned from a 2.5-year hiatus, does anyone think it should still support Rails 2.x? Please share your opinion.

**Comment by dnrce** [comment]
Closing this as "wont fix". Sorry, Rails 2, but you're old.

Anyone needing Rails 2 support: feel free to make noise here in the future, but it may or may not get attention. You're probably better off trying an old version of yaml_db (0.2.x).

**Comment by czak** [comment]
Let me share my story of the `Rails2` branch.

A few weeks ago I reached a 10,000 free DB rows limit on Heroku, prompting me to either pay up or move out. This was for a Rails 2.3 app I wrote between 2009-2011. The app was consistently used by one person all these years so I was not going to abandon it, but I wasn't willing to invest much time or money into it.

I decided to dockerize it and - to keep a minimal footprint - opted for sqlite. This works fine.

Now for the fun part: I was glad to read about `yaml_db`'s `Rails2` branch. It worked without a glitch as a plugin (yeah `vendor/plugins/` ... that brought some memories) and allowed me to migrate postgresql &rarr; sqlite in a few minutes.

So here's to you `Rails2` branch, you have served me well. Sad to see you go :wine_glass:
